recipe-server/client/control_new/state/recipes/selectors.js

```diff
@@ -25,6 +26,14 @@ export function getRecipe(state, id, defaultsTo = null) {
   return defaultsTo;
 }
 
+export function getCurrentRecipePk(state) {
```

Comment:
I ditched all the `currentRecipe` stuff which you can also see in the recipe details PR.